applications/ironside/se/update/README.rst

```diff
@@ -0,0 +1,56 @@
+.. _ironside_se_update_application:
+
+IRONside SE Update application
+##############################
+
+This application is used to trigger an update of the IRONside SE firmware.
+The new firmware can be found in the release zip of the IRONside SE.
+Both recovery and main IRONside SE firmwares can be updated.
+
+Update procedure
+****************
+
+IRONside SE firmware update hex files are located at a known address.
+This application invokes the IRONside SE update service, passing the parameters corresponding to the update hex file.
+The return value of the service call is printed, as well as information from the update hex file.
+Once the service has been invoked, the IRONside SE firmware has updated the internal state of the device so that the update will be installed on the next boot.
+
+Requirements
+************
+
+The firmware supports the following development kit:
+
+.. table-from-sample-yaml::
+
+
+Building and running
+********************
+
+This application can only be used when there is already a version of IRONside SE installed on the device.
```

Comment:
```suggestion
.. note::
   You can use this application only when a version of IRONside SE is already installed on the device.
```

Reply:
@hakonfam Can you build and program this app using VScode?

Reply:
The IRONside SE app? No.

Reply:
Please add this note